.gitignore

```diff
@@ -2,8 +2,4 @@
 build/
 website/public
 .envrc
-e2etest/vmlinuz
-e2etest/initrd.img
-e2etest/*.qcow2
-e2etest/e2etest
-e2etest-cached-universe
+kind.yaml
```

Comment:
Did this get auto-created?  Or is it an optional thing that people might create?  I just noticed I don't have this file.

Reply:
I just realized it was there from a couple of months ago when I was looking at this repo. I ran "dev-env-cleanup" and didn't check if that was cleaned up, and hence part of, the "dev-env" task. I have removed that.